sunpy/net/dataretriever/sources/swap.py

```diff
@@ -59,12 +58,18 @@ def _get_url_for_timerange(self, timerange, **kwargs):
         returns list of urls corresponding to given TimeRange.
         """
         level = kwargs.get('level', 1)
-        SWAP_STARTDATE = datetime.datetime(2009, 11, 24)
+        if (level == 1 or level == 0):
+            SWAP_STARTDATE = datetime.datetime(2009, 11, 24)
+        else:
+            SWAP_STARTDATE = datetime.datetime(2010, 1, 4)
         if timerange.start < SWAP_STARTDATE:
             raise ValueError('Earliest date for which SWAP data is available is {:%Y-%m-%d}'.format(SWAP_STARTDATE))
-        datatype = {0: 'eng', 1:'bsd'}
-        prefix = 'http://proba2.oma.be/swap/data/{datatype}/'
-        suffix = '%Y/%m/%d/{instrument}_lv{level}_%Y%m%d_%H%M%S.fits'
+        datatype = {0: 'eng', 1:'bsd', 'q':'qlk', 'Q':'qlk'}
```

Comment:
You could just use `'q': 'qlk'` and compare with the lowercase of the input. However I'm not too convince to get an argument that accepts numbers and strings... @wafels - what do you think?